scripts/requirements-darwin.txt

```diff
@@ -0,0 +1,9 @@
+python=3.7.7;hash=b703318dd9eb7bc5e8d97c8b95e9445f754ca22f
```

Comment:
This hash belongs to python 3.8.2.

Reply:
thanks, and correct.
Not sure how that happened to be wrong 😕

Reply:
fixed